src/sst/core/simulation_impl.h

```diff
@@ -603,6 +604,9 @@ void wait_my_turn_start(Core::ThreadSafe::Barrier& barrier, int thread, int tota
 
 void wait_my_turn_end(Core::ThreadSafe::Barrier& barrier, int thread, int total_threads);
 
+// Function to exit, guarding against race conditions if multiple threads call it
```

Comment:
Looks like this function needs to be hoisted above the uses of it in the same file.  This is causing all the testing builds to fail.